src/OrchardCore.Modules/Orchard.Roles/Services/RoleStore.cs

```diff
@@ -67,8 +67,6 @@ public void UpdateRoles(RolesDocument roles)
         #region IRoleStore<Role>
         public async Task<IdentityResult> CreateAsync(Role role, CancellationToken cancellationToken)
         {
-            cancellationToken.ThrowIfCancellationRequested();
```

Comment:
> we should remove that parameter

This is part of the EF interface

> whats with the removal of this

When I implemented the provider I reflected what the EF ones where doing. But when doing perf tests I would often see errors in the log because this exception was thrown. What happens is that Identity passes the Kestrel CancellationToken to its services, and it's wrong. This means if a connection is aborted from the client, this token will be true, and instead of running the following code it will just throw. I can tell you that people will ask again and again why their logs are full of these exceptions.

We are free to do what we want with this token. I think we should just go on and let the request finish, instead of throwing. We can also return an empty result to shortcut, but there is a lot of thing that can happen after this call, and we are not handling the cancellation tokens elsewhere so why bother doing it in a very small part like this.

My conclusion is to just ignore this token. We don't need to use it.